package.json

```diff
@@ -92,5 +94,8 @@
     "stylelint": "^13.7.2",
     "stylelint-config-standard": "^20.0.0",
     "stylelint-value-no-unknown-custom-properties": "^3.0.0"
+  },
+  "resolutions": {
+    "jsdom": "16.3.0"
```

Comment:
Addition of this was a fix to a problem where jest + the new react-testing-library caused weird console warnings during test runs.